lib/shopify_transporter/pipeline/magento/order/line_items.rb

```diff
@@ -52,6 +57,42 @@ def tax_lines(item)
             ]
           end
 
+          def total_discount(item)
+            item['discount_amount'] if item['discount_amount'].present? && item['discount_amount'].to_f != 0
```

Comment:
Does this actually ever work? (followup to our IRL discussion yesterday afternoon)

We shouldn't merge this if it never does anything. We should experiment with the behaviour of this column/spike into it.

Reply:
All the discount-related fields are not working as of now. I've implemented the code on the app side to prepare and send discount data, but the Shopify API is setting ``total_discounts``(top level), ``total_discount``(line item level) and ``discount_amount`` (top level) all to 0 for some reason. 

I had a conversion with @surajreddy offline. He stated that he was aware of such behaviour of the Shopify API, the code related to ``total_discounts`` in the app existed but then was removed for this reason. I'd like to get more context from Suraj after he is free from the delete incident. For now I will hold off on any implementation related to discount stuff.